pages/+layout.svelte

```diff
@@ -0,0 +1,9 @@
+<script>
+  import { EvidenceDefaultLayout } from '@evidence-dev/core-components';
+  import site from '$lib/config/site';
+  import '@evidence-dev/core-components/global.css';
+</script>
+
+<EvidenceDefaultLayout githubRepo="https://github.com/your-repo-url">
```

Comment:
The repo url is archiewood/baby-tracker

Reply:
Are you going to address this?

Reply:
The repo name is github.com/archiewood/baby-tracker

You also need to pass the {data} prop to the EvidenceDefaultLayout